packages/rum-core/src/error-logging/error-logging.js

```diff
@@ -110,12 +110,31 @@ class ErrorLogging {
     }
   }
 
-  registerGlobalEventListener() {
+  onUnhandledError() {
     window.addEventListener('error', errorEvent =>
       this.logErrorEvent(errorEvent)
     )
   }
 
+  onUnhandledRejection() {
+    window.addEventListener('unhandledrejection', promiseRejectionEvent =>
+      this.logPromiseEvent(promiseRejectionEvent)
+    )
+  }
+
+  logPromiseEvent(promiseRejectionEvent) {
+    const prefix = 'Unhandled promise rejection: '
+    const { reason } = promiseRejectionEvent
+
+    if (reason == null) {
```

Comment:
Do we need to check for undefined?

Reply:
`==` checks for both `null` and `undefined`.

Reply:
What about other falsy values?